### ISSUE TITLE
Class Teams Require Teacher to Activate Team Before Students Can Access

### ISSUE THREAD

**Issue by thebenkahn** [microsoft/o365-moodle#2138]
Hi Lai Wei

I have a question/potential feature enhancement. We have a client that utilizes the Moodle/Teams sync heavily. They use the "All Features Enabled" option in course sync settings to create a Team for all courses. Recently they noticed that the new Teams created require an owner to go in and [Activate the Team before students can use it](url). This is a problem because there teachers are more like facilitators and it is expected that students can jump in the Team right away. So they ended up with an LMS admin/Teams owner going through and manually activating several hundred courses.

I understand from the release notes that the plugin was changed to check the connected Azure tenant and enforce that associated Teams should be created as Class Teams if it is an EDU tenant, and that this is following the MS API guidelines. So I do not expect to be able to specify that we should create standard Teams in this environment.

My thought is, is it possible to bypass the Activate requirement in the graph API when creating the Team and if so can that be a configurable option in the Teams sync settings? I looked through the API documentation relevant to creating a Team from a template and didn't see anything obvious but thought it worth asking the expert.

If not, are there any resources I can point people to on how to configure this on the Azure side or to script activating in bulk besides "reach out to your Microsoft rep?"

Thanks so much!

**Comment by weilai-irl** [comment]
Hi @thebenkahn,

You are not alone. This was among the first questions that asked the Microsoft education team when I saw the guidelines, as I saw this could be a major change in the workflow for many. I was informed that this would be the standard workflow for all class teams, and yes, all class teams need to be manually activated by team owners before they can be accessed by members.

I have found two documentations from Microsoft about the requirement of activating class Teams:
 - https://support.microsoft.com/en-us/topic/create-a-class-team-in-microsoft-teams-fae422eb-58b7-4431-9ff2-a4b9b6ae7c5b
 - https://support.microsoft.com/en-us/topic/activate-early-access-class-teams-created-by-your-it-admin-0d154696-66ab-4fcf-b22f-c3d9a82aaf78

I hope this helps.

Regards,
Lai
